src/routes/blog/posts/v3-1/+page.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Announcing Files v3.1
+description: System tray icon, open minimized on startup, Smart Extract, prioritize files when sorting, and a new drag & drop gesture.
+thumbnail: /blog-resources/v3-1/Hero.jpg
+date: 1/9/2024
+author: files-community
+---
+
+We're excited to announce that starting today, v3.1 is now available. If you already have Files, there should be a notification on the toolbar to install the update. Otherwise, you can download Files for free from our [download page](/download/). You can also purchase Files on the [Microsoft Store](ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite) or sponsor us on [GitHub](https://github.com/sponsors/yaira2) to help support the project.
```

Comment:
How about "Files v3.1 is now available"? Having only v3.1 seems a bit strange but maybe that's just me.

Reply:
👍